.github/actions/ci/action.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+
+description: Shared action to install dependencies
+
+runs:
+  using: composite
+
+  steps:
+    - name: Read .nvmrc
```

Comment:
would it make sense to add the checkout step here also, as it would ensure that the .nvmrc is present

Reply:
Not really, it is not necessary here as the repo is already checked-out in the step before. Also we are only just reading out the `.nvmrc` and making the version within available in the environment.